### ISSUE TITLE
[DOCS] Make the hosted link clickable in the README

### ISSUE THREAD

**Issue by Pradumnasaraf** [Pradumnasaraf/Post-My-Message#8]
### Description

Remove the backticks from the text and make it clickable

it will look something like this 

```markdown
[https://post-my-message.vercel.app](https://post-my-message.vercel.app/)
```

### Screenshots

<img width="854" alt="Screenshot 2023-02-10 at 12 41 28 PM" src="https://user-images.githubusercontent.com/51878265/218026245-9ea0fb49-0414-49c3-90eb-b799c284e814.png">

**Comment by Abhishek7Tech** [comment]
@Pradumnasaraf Hi I would like to contribute

**Comment by Pradumnasaraf** [comment]
Go ahead.

**Comment by shryshs** [comment]
@Pradumnasaraf  can I also raise a PR on this issue?

**Comment by Pradumnasaraf** [comment]
Hey, @ShreyashSomvanshi it's already assigned to someone, so you can't work on that. TRy finding some other issue or open it if you find something

**Comment by Abhishek7Tech** [comment]
@Pradumnasaraf Let @ShreyashSomvanshi  contribute he has already done the work!.

**Comment by Pradumnasaraf** [comment]
> @Pradumnasaraf Let @ShreyashSomvanshi contribute he has already done the work!.

No worries you can go ahead

**Comment by Abhishek7Tech** [comment]
@Pradumnasaraf Working on it!

**Comment by gauravrattan** [comment]
@Pradumnasaraf assign me

**Comment by AyonChowdhury1307** [comment]
If no one is working on this issue, I would like to contribute to this

**Comment by Pradumnasaraf** [comment]
@Abhishek7Tech is already working look for other open issues

**Comment by Pradumnasaraf** [comment]
fix #40
